.github/workflows/deploy.yml

```diff
@@ -143,3 +143,22 @@ jobs:
           body_path: RELEASE.md
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  aur_release:
+    runs-on: ubuntu-latest
+    name: Create AUR release
+    needs: github_release # The -bin version downloads and shasums it
+    steps:
+      - name: Build | Publish AUR package
+        uses: guumaster/aur-publish-docker-action@v1
+        with:
+          package_name: starship
+          commit_username: "Starship Bot"
+          commit_email: starship.bot.noreply@gmail.com
+          ssh_private_key: ${{ secrets.AUR_SSH_PRIVATE_KEY }}
+      - name: Build | Publish AUR package (-bin version)
+        uses: guumaster/aur-publish-docker-action@v1
+        with:
+          package_name: starship-bin
+          commit_username: "Starship Bot"
+          commit_email: starship.bot.noreply@gmail.com
```

Comment:
Is it a problem if the email isn't monitored? I'm going out on a limb and assuming that very few people use the committer email in the AUR git repo as information on how to contact the maintainer.

Reply:
They shouldn't. If they want to contact us they should use the comments in the AUR web ui or create an issue here on GitHub.